javagen/src/main/java/com/azure/autorest/mapper/ClientMethodMapper.java

```diff
@@ -894,21 +896,19 @@ protected static String returnTypeDescription(Operation operation, IType returnT
                 }
             }
 
-            if (description == null) {
-                if (baseType == PrimitiveType.Void) {
-                    // Mono<Void>
-                    description = "the completion";
-                }
-                if (baseType == PrimitiveType.Boolean
-                        && operation.getRequests() != null && !operation.getRequests().isEmpty()
-                        && operation.getRequests().get(0).getProtocol() != null
-                        && operation.getRequests().get(0).getProtocol().getHttp() != null
-                        && HttpMethod.HEAD.name().equalsIgnoreCase(operation.getRequests().get(0).getProtocol().getHttp().getMethod())) {
-                    // Mono<Boolean> of HEAD method
-                    description = "whether resource exists";
-                }
+            if (description == null
+                && baseType == PrimitiveType.Boolean
+                && operation.getRequests() != null && !operation.getRequests().isEmpty()
+                && operation.getRequests().get(0).getProtocol() != null
+                && operation.getRequests().get(0).getProtocol().getHttp() != null
+                && HttpMethod.HEAD.name().equalsIgnoreCase(operation.getRequests().get(0).getProtocol().getHttp().getMethod())) {
+                // Mono<Boolean> of HEAD method
+                description = "whether resource exists";
+                return description;
```

Comment:
Personally I would try avoid `return` in middle of method. It is kind of `goto`, and make the logic hard to understand.